griptape/drivers/vector/local_vector_store_driver.py

```diff
@@ -19,7 +19,7 @@
 class LocalVectorStoreDriver(BaseVectorStoreDriver):
     entries: dict[str, BaseVectorStoreDriver.Entry] = field(factory=dict)
     persist_file: Optional[str] = field(default=None)
-    relatedness_fn: Callable = field(default=lambda x, y: dot(x, y) / (norm(x) * norm(y)))
+    relatedness_calculator: Callable = field(default=lambda x, y: dot(x, y) / (norm(x) * norm(y)))
```

Comment:
Why no generic params?

Reply:
They should be there but I want to keep this PR strictly on naming.